roles/byzantine/defaults/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# defaults file for roles/byzantine
+apache_app_path: '{{ drupal_docroot }}/current'
+capistrano_base_dir: '{{ drupal_base_dir }}'
+capistrano_web_owner: '{{ drupal_web_owner }}'
+datadog_trace_environment: ''
+ddtrace_repo_version: '0.56.0'
+deploy_user: 'deploy'
+drupal_base_dir: '/var/www'
+drupal_base_path: 'http://localhost'
+drupal_db_name: 'byzantine'
+drupal_db_user: 'byzantine'
+drupal_db_password: "{{ libwww_db_password | default('change_this') }}"
+drupal_db_port: '3306'
+drupal_key_file: '/home/deploy/.ssh/id_rsa'
+drupal_local_repo: ''
+drupal_major_version: 7
+drupal_project: 'byzantine'
+drupal_redis_host: 'localhost'
+drupal_redis_prefix: 'byzantine'
+drupal_ssl_base_path: 'https://localhost'
+drupal_web_owner: 'www-data'
+force_settings: false
+install_mailcatcher: false
+smtp_host: 's:26:"lib-ponyexpr.princeton.edu";'
```

Comment:
Hrm, good point. Lemme ponder.

Reply:
I think I'd rather leave it - the smtp_host needs to be there in either environment (it's just that mailcatcher will catch it), and since the defaults are largely for CI, I don't want to add unnecessary complexity by adding in mailcatcher to the CI runs. 

I'm open to being convinced, though.